### PR TITLE
Code Cleanup

### PR DESCRIPTION
**Pull request by Christian-Oleson**

### CODE REVIEW

File: src/SerialPortComms.cs [hunk 1]
```diff
@@ -1,54 +1,50 @@
...
-using System.Text;
 using System.Threading;
-using System.Threading.Tasks;
 
 namespace COMSpike
 {
     public class SerialPortComms
     {
-        private SerialPort port;
+        private readonly SerialPort _port;
```
Comment (nekkid-yoga) [inline src/SerialPortComms.cs:9]: Typically you'd use the target naming conventions for a PR... but this is fine 🤣

File: src/SerialPortComms.cs [hunk 2]
```diff
@@ -1,54 +1,50 @@
...
         {
-            port.Open();
+            _port.Open();
 
-            Console.WriteLine($"IsOpen: {port.IsOpen}");
+            Console.WriteLine($"IsOpen: {_port.IsOpen}");
 
-            Console.WriteLine("Writing 'Reboot' to port...");
-            port.Write("Reboot");
+            Console.WriteLine("Writing 'Reboot' to _port...");
```
Comment (nekkid-yoga) [inline src/SerialPortComms.cs]: Search and replace gone wrong

File: src/SerialPortComms.cs [hunk 3]
```diff
@@ -1,54 +1,50 @@
...
 
             Close();
         }
 
-        internal void Close()
+        private void Close()
         {
-            Console.WriteLine("Closing port...");
-            port.Close();
+            Console.WriteLine("Closing _port...");
```
Comment (nekkid-yoga) [inline src/SerialPortComms.cs]: Search and replace gone wrong

File: src/SerialPortComms.cs [hunk 4]
```diff
@@ -1,54 +1,50 @@
...
+        private static void Port_ErrorReceived(object sender, SerialErrorReceivedEventArgs e)
         {
             Console.WriteLine($"Error Received: {e.EventType}");
         }
 
         private void Port_DataReceived(object sender, SerialDataReceivedEventArgs e)
         {
-            // Show all the incoming data in the port's buffer
-            Console.WriteLine($"Data Received: {port.ReadExisting()}");
+            // Show all the incoming data in the _port's buffer
```
Comment (nekkid-yoga) [inline src/SerialPortComms.cs]: Search and replace gone wrong

**Review by nekkid-yoga — CHANGES_REQUESTED** [review]

File: src/SerialPortComms.cs (hunk 1 above)
Comment (Christian-Oleson) [reply src/SerialPortComms.cs:9]: I didn't follow that, sorry.

File: src/SerialPortComms.cs (hunk 4 above)
Comment (Christian-Oleson) [reply src/SerialPortComms.cs]: whoops, I trusted VS 2022 to do the job. 😆

**Review by nekkid-yoga — APPROVED** [review]

**Pull request merged**